### DISCUSSION TITLE
Change Weblate domain

### DISCUSSION THREAD

**Discussion by ghost** [WeblateOrg/weblate#10013] [Q&A] [answered]
Hello,

We would like to change the URL of our Weblate instance. Is it right to assume that:
1. We need to change the `SITE_DOMAIN` variable as instructed [in the documentation](https://docs.weblate.org/en/latest/admin/install.html#set-correct-site-domain) first
2. We then need to update our reverse proxy (Nginx in our case) with the new domain name and the corresponding TLS certificates
3. Reload Nginx configuration and restart the Weblate Docker container

Am I not overlooking something?

Thanks in advance for your help!

**Comment by nijel** [comment, marked as answer]
I don't think anything else would be needed.

**Reply by ghost** [reply]
Thank you for your answer! When doing it, the Weblate repository URLs in **Repository maintenance** are still the old ones!

**Reply by ghost** [reply]
I found a workaround: I have to go to the component settings > Version control and just hit the Save button at the end of the page. Then the URLs are updated and matches the config!

For your information I also had to update the `WEBLATE_ALLOWED_HOSTS` variable to add the new domain name.

Thanks again for your help!

**Reply by nijel** [reply]
WEBLATE_ALLOWED_HOSTS defaults to *, there is typically no need to set it up when running behind a reverse proxy.
